### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Saywin**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1,8 +1,20 @@
...
 
-def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+
+def create_person_list(people: list) -> list[Person]:
+    for person in people:
+        Person(person["name"], person["age"])
+    for human in people:
+        if "wife" in human and not human["wife"] is None:
```
[the remark this answers is not in the document]
Comment (Saywin) [reply app/main.py]: Thank you, I have not used this method before

File: app/main.py [hunk 2]
```diff
@@ -1,8 +1,20 @@
...
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
-def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+
+def create_person_list(people: list) -> list[Person]:
```
Comment (vasylhnatiuk) [inline app/main.py:10]:
```suggestion
def create_person_list(people: list[dict]) -> list[Person]:
```

File: app/main.py [hunk 3]
```diff
@@ -1,8 +1,20 @@
...
+    for person in people:
+        Person(person["name"], person["age"])
+    for human in people:
+        if human.get("wife") is not None:
+            Person.people[human["name"]].wife \
```
Comment (vasylhnatiuk) [inline app/main.py:15]: try to avoid \

**Review by vasylhnatiuk — CHANGES_REQUESTED** [review]

**Review by vsmutok — APPROVED** [review]
Looks Good!

**Comment by Saywin** [comment]
Thanks for checking

**Pull request closed without merging**